tests/modules/ichorcna/createpon/main.nf

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { ICHORCNA_CREATEPON } from '../../../../modules/ichorcna/createpon/main.nf'
+
+workflow test_ichorcna_createpon {
+
+    input = file("https://raw.githubusercontent.com/broadinstitute/ichorCNA/master/inst/extdata/MBC_315.ctDNA.reads.wig", checkIfExists: true)
```

Comment:
If there is no usable test data in nf-core/test-datasets `modules` branch, then you should add it there prefereably based on data that is already there. The paths are then added [here](https://github.com/nf-core/modules/blob/master/tests/config/test_data.config) and you can access them using keys (see [here](https://github.com/nf-core/modules/blob/master/tests/modules/abacas/main.nf))

Reply:
As far as I can tell, there is nothing useful on the test-datasets at the moment. It requires reads across more of the genome, not just a small fraction of one chromosome. If I try the test human datasets it just crashes, and it gives an error on the covid genome related to trying to look up the genome details annoyingly.

Reply:
@GCJMackenzie has added also reads mapping to chr21. Would it be enough to combin chr22 and chr21 reads we have currently?

Reply:
Thinking about it, I can specify the chromosomes to consider in the tool, so that may be sufficient. Which file is that?

Reply:
Part of the problem is that the test data is completely fake, for instance it is mapped to regions that are actually telomeres.

Reply:
For future refernce: as an exception the github uirls are used here: https://nfcore.slack.com/archives/CJRH30T6V/p1642677700058500